src/cltkv1/morphology/Morphosyntax.py

```diff
@@ -0,0 +1,330 @@
+"""A module for representing universal morphosyntactic feature bundles."""

```

Comment:
A Python filename [should begin with a lowercase character](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).

Reply:
Ugh forgot that.  Thanks.